src/mumble/MainWindow.cpp

```diff
@@ -2531,6 +2533,34 @@ void MainWindow::on_qaChannelCopyURL_triggered() {
 										   QClipboard::Clipboard);
 }
 
+void MainWindow::on_qaChannelDescriptionView_triggered() {
+	Channel *c = getContextMenuChannel();
+	// This has to be done here because UserModel could've set it.
+	cContextChannel.clear();
+
+	if (!c)
+		return;
+
+	if (!c->qbaDescHash.isEmpty() && c->qsDesc.isEmpty()) {
+		c->qsDesc = QString::fromUtf8(Global::get().db->blob(c->qbaDescHash));
+		if (c->qsDesc.isEmpty()) {
+			pmModel->iChannelDescription = ~static_cast< int >(c->iId);
```

Comment:
Just for the record, I think here the same issue with the root channel applies that Hartmnt has found elsewhere already.